### ISSUE TITLE
Can't switch "app-switcher" to only "super" key.

### ISSUE THREAD

**Issue by NeuHix** [G-dH/advanced-alttab-window-switcher#53]
Can you tell me how to invoke the app switcher by pressing the window key (super) ?

Currently it is Win+Tab.

I'm in Fedora 36 Gnome Wayland.

I didn't found that setting in the Gnome Tweaks app.

**Comment by G-dH** [comment]
AATWS has option _Super Key Action_ on the _Common_ page of its Settings window, it replaces the default overview action.
In GNOME Tweaks you can change the _Super_ key to _Alt_ or disable it at all.
If you want to disable the _Alt+Tab_ shortcut, use _GNOME Settings_ > _Keyboard Shortcuts_: https://github.com/G-dH/advanced-alttab-window-switcher/issues/23#issuecomment-1111930055.

**Comment by NeuHix** [comment]
Changing the Overview Shortcut to Left Super in the Gnome Tweaks worked.
I also disabled the "switch applications" shortcut.

Thanks! For the help. 
Closing the issue.


![Screenshot from 2022-12-16 03-29-21](https://user-images.githubusercontent.com/94163435/207976162-f5ba4061-e13f-44cc-97e8-bd2858732993.png)
![image](https://user-images.githubusercontent.com/94163435/207976584-b9dbddc9-bf72-4026-b36d-e3bb7cd35e7d.png)
![Screenshot from 2022-12-16 03-29-09](https://user-images.githubusercontent.com/94163435/207976680-a4f44bb8-a88f-4269-be36-0c6172baeb4a.png)
